be/src/common/config.h

```diff
@@ -557,7 +557,7 @@ CONF_mInt64(storage_flood_stage_left_capacity_bytes, "107374182400"); // 100GB
 CONF_mDouble(storage_high_usage_disk_protect_ratio, "0.1"); // 10%
 
 // Number of thread for flushing memtable per store.
-CONF_mInt32(flush_thread_num_per_store, "2");
+CONF_mInt32(flush_thread_num_per_store, "4");
```

Comment:
Could we determine the number by the disk type?

Reply:
Good idea! But we can only determine if the disk is HDD or SSD, but in face, the cloud SSD such as ESSD on AliCloud, their latency is much more high than local SSD, so it may need more flush thread than local SSD.